client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import _orderBy from 'lodash/orderBy';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
+import store from 'store';
+import { Table } from '../../components/generic';
+
+const columns = [
+  { id: 'pptId', name: 'ID' },
```

Comment:
Nitpick: I think we've been avoiding most ambiguous acronyms and shortenings. Don't think we're using `ppt` for participant anywhere else.